web/modules/custom/joinup_subscription/joinup_subscription.module

```diff
@@ -86,9 +86,17 @@ function joinup_subscription_message_digest_aggregate_alter(array &$context, Use
  *
  * Performs the following alterations:
  * - Sends digest mails as HTML.
+ * - Changes the mail subject.
  */
 function joinup_subscription_mail_alter(array &$message): void {
   if ($message['key'] === 'digest') {
     $message['headers']['Content-Type'] = SWIFTMAILER_FORMAT_HTML;
+    $notifier = $message['params']['notifier'] ?? NULL;
+    if ($notifier instanceof MessageNotifierInterface) {
+      $interval = $notifier->getPluginDefinition()['id'];
+      $message['subject'] = t('Joinup: @interval digest message', [
+        '@interval' => $interval,
+      ]);
```

Comment:
This wrong. The plugin ID is not translatable and it's an ID, it may have a value that has no meaning for a human. We should use the plugin's title as that is extracted from the `message_digest_interval` config entity and being a label is translatable. See `DigestDeriver::getDerivativeDefinitions()`.